packages/core/src/Stores/ui-store.js

```diff
@@ -264,6 +299,10 @@ export default class UIStore extends BaseStore {
     @action.bound
     closeRealAccountSignup() {
         this.is_real_acc_signup_on = false;
+        setTimeout(() => {
+            this.resetRealAccountSignupParams();
+            this.setRealAccountSignupEnd(true);
+        }, 300);
```

Comment:
is timeout necessary here?

Reply:
@kelly-binary Thanks - actually, I asked @reza-fs about the flow, and he mentioned that it isn't, but I had not referred him to this line. @reza-fs Can you confirm this one is unnecessary too please?

Reply:
if we reset the `accountSignupParams` too early, the user will see a flash of the first modal page. So, the thing we talked about wasn't here (Or I was looking at somewhere else at the time :joy:).

Reply:
@reza-fs @kelly-binary So we shall keep this.

And @reza-fs yes, the initial link I sent to you was the origin of the timeout, then I moved the content of that and put it inside this one, so I wanted to ask if we have to keep it or do I remove it altogether.